app/models/comments.models.js

```diff
@@ -0,0 +1,15 @@
+const db = require("/Users/tiahhind/Northcoders/backend/be-nc-news/db/connection.js");
```

Comment:
If you look at this file path, would this work if someone else was to run the code?

Reply:
Was having issues getting relative filepath but this has now been fixed by moving connection.js out of db folder